shared/db/userdb.go

```diff
@@ -124,3 +124,28 @@ func GetHash(id int) string {
 
 	return ""
 }
+
+func IsParticipant(userID int, courseID int) bool {
```

Comment:
This function already exist in course.db as UserExistsInCourse, either remove the old one or switch

Reply:
fixd